### ISSUE TITLE
Sporadic failing unit test in CI TestNormalizeTx1

### ISSUE THREAD

**Issue by catenocrypt** [trustwallet/blockatlas#1111]
I saw a unit test failing sporadically in CI.  TestNormalizeTx1.  Apparently it includes a timestamp which may change.  Sample failing CI:

https://github.com/trustwallet/blockatlas/pull/1110/checks?check_run_id=690404584

Sample log:

```
2020-05-19T20:03:23.0101153Z === RUN   TestNormalizeTx1
2020-05-19T20:03:23.0101774Z === RUN   TestNormalizeTx1/test_transaction
2020-05-19T20:03:23.0102089Z === RUN   TestNormalizeTx1/test_pending_transaction
2020-05-19T20:03:23.0102412Z     TestNormalizeTx1/test_pending_transaction: transaction_test.go:99: 
2020-05-19T20:03:23.0102740Z         	Error Trace:	transaction_test.go:99
2020-05-19T20:03:23.0103043Z         	Error:      	Not equal: 
2020-05-19T20:03:23.0103721Z         	            	expected: blockatlas.Tx{ID:"79719d16f3f347cc98c35cd7a9af708cdce97de578b5135c5ae4393fd7920d61", Coin:0xf2, From:"NQ74 SJ0Q 49T1 4XQL KABH 1RUC 8DPT 9F0U 9P0B", To:"NQ97 18BJ 33YV QGHQ BV2K 56V4 12CH J8TD S9S3", Fee:"300", Date:1589918602, Block:0x0, Status:"", Error:"", Sequence:0x0, Type:"", Inputs:[]blockatlas.TxOutput(nil), Outputs:[]blockatlas.TxOutput(nil), Direction:"", Memo:"", Meta:blockatlas.Transfer{Value:"100000", Symbol:"NIM", Decimals:0x5}}
2020-05-19T20:03:23.0104229Z         	            	actual  : blockatlas.Tx{ID:"79719d16f3f347cc98c35cd7a9af708cdce97de578b5135c5ae4393fd7920d61", Coin:0xf2, From:"NQ74 SJ0Q 49T1 4XQL KABH 1RUC 8DPT 9F0U 9P0B", To:"NQ97 18BJ 33YV QGHQ BV2K 56V4 12CH J8TD S9S3", Fee:"300", Date:1589918603, Block:0x0, Status:"", Error:"", Sequence:0x0, Type:"", Inputs:[]blockatlas.TxOutput(nil), Outputs:[]blockatlas.TxOutput(nil), Direction:"", Memo:"", Meta:blockatlas.Transfer{Value:"100000", Symbol:"NIM", Decimals:0x5}}
2020-05-19T20:03:23.0104711Z         	            	
2020-05-19T20:03:23.0106463Z         	            	Diff:
2020-05-19T20:03:23.0107668Z         	            	--- Expected
2020-05-19T20:03:23.0108107Z         	            	+++ Actual
2020-05-19T20:03:23.0108714Z         	            	@@ -6,3 +6,3 @@
2020-05-19T20:03:23.0109149Z         	            	  Fee: (blockatlas.Amount) (len=3) "300",
2020-05-19T20:03:23.0109754Z         	            	- Date: (int64) 1589918602,
2020-05-19T20:03:23.0110191Z         	            	+ Date: (int64) 1589918603,
2020-05-19T20:03:23.0110525Z         	            	  Block: (uint64) 0,
2020-05-19T20:03:23.0110963Z         	Test:       	TestNormalizeTx1/test_pending_transaction
2020-05-19T20:03:23.0111471Z --- FAIL: TestNormalizeTx1 (0.00s)
2020-05-19T20:03:23.0112150Z     --- PASS: TestNormalizeTx1/test_transaction (0.00s)
2020-05-19T20:03:23.0112747Z     --- FAIL: TestNormalizeTx1/test_pending_transaction (0.00s)
```

**Comment by catenocrypt** [comment]
Nimiq chain; NormalizeTx uses current time for pending Txs, and expected data has also current time.  However, with bad luck the two can be 1 sec apart.

Date:  time.Now().Unix(),

**Comment by catenocrypt** [comment]
Being solved in #1110

**Comment by catenocrypt** [comment]
Solved with #1110
